sycl/ReleaseNotes.md

```diff
@@ -2,20 +2,610 @@
+
+### Explicit SIMD extension
+
+- Extended
+  [`sycl_ext_intel_esimd`](https://github.com/intel/llvm/blob/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/doc/extensions/supported/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md)
+  extension specification and implementation with new queries to check support
+  for 2d load/store/prefetch operations. intel/llvm#15905
+- Fixed miscompilations of ESIMD functions under high optimization levels when
+  compiler performs aggressive inlining. intel/llvm#16193
+
+### Sanitizers
+
+- ada16682e8c3 [DevASAN] Only report warning if passing host ptr to kernel (#16654)
+  - seems like a potentially important bugfix
+- ce4a320806b2 [DeivceASAN] Make ShadowMemory one instance per type (#16687)
+  - seems like some kind of bugfix
+- ef4d66af3b74 [DeviceSAN] Fix kernel name addressspace (#16425)
+- 1fba00d3be7d [DeviceASAN] Fix ASAN with kernel assert (#16256)
+- 34aeabab551e [SYCL][DeviceASAN] Fix AcceeChain to a matrix for bfloat16 (#16323)
+- 6f3b0e857d15 [DevASAN] Do allocation with USM pool to reduce memory overhead (#16280)
+- a8c6e7715be2 [DeviceASAN] Re-use shadow if required size is not larger than last one (#16258)
+  - As the above, some optimization of memory usage by ASAN?
+- 201725664cc5 [DeviceSanitizer] Fix device global type of KernelMetadata (#16357)
+
+#### Address Sanitizer
+
+- Fixed ASAN throwing an exception with `UR_RESULT_ERROR_INVALID_ARGUMENT` when
+  detecting incorect memory free operation. intel/llvm#16706
+
+### Bindless images
+
+- Added support for `ext_oneapi_bindless_sampled_image_fetch_1d`,
+  `ext_oneapi_bindless_sampled_image_fetch_1d_usm`,
+  `ext_oneapi_bindless_sampled_image_fetch_2d`,
+  `ext_oneapi_bindless_sampled_image_fetch_2d_usm` and
+  `ext_oneapi_bindless_sampled_image_fetch_3d` aspects on Level Zero backend.
+  intel/llvm#16862
+- Added the initial support for bindless images on AMD GPUs. intel/llvm#16439
+- Fixed return types of image extent queries to match the specification.
+  intel/llvm#16829
+- Clarified the types of supported USM memory in the extension specification.
+  intel/llvm#16622
+- Fixed compiler crash caused by the use of anisotropic sampling operations on 3D mipmaps,
+  due to the intrinsic being generated with an incorrect number of LOD gradient parameters.
+  intel/llvm#16135
+
+### Native CPU device
+
+- Improved support for `dynamic_address_cast` on Native CPU device.
+  intel/llvm#16676
+- Improved performance of Native CPU device: less memory allocations and thread
+  launches. intel/llvm#17102
+- Fixed a bug where submitting the same kernel multiple times at about the same
+  time with different argument would lead to incorrect arguments being used.
+  intel/llvm#16995
+- Fixed compiler crashes when building applications that use atomics.
+  intel/llvm#16737
+- Fixed segfaults happening in SYCL CTS tests for `async_work_group_copy`
+  API. intel/llvm#16500
+- Improved support for sub-groups by updating version of OneAPI Construction
```

Comment:
```suggestion
- Improved support for sub-groups by updating version of oneAPI Construction
```

Reply:
Let me also just mark it as approved already aside from this, so that it doesn't hold up anything.